src/components/Section/Identification/Identification.jsx

```diff
@@ -21,22 +23,51 @@ import Review from 'components/Section/Identification/Review'
 
 /**
  * TODO
- * - subsection prop is not defaulting to "intro" after login, this is prob related to keeping redux in sync with routes. Investigate & fix this.
+ * - subsection prop is not defaulting to "intro" after login.
+ * this is prob related to keeping redux in sync with routes. Investigate & fix this.
  */
 
 class Identification extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.form = formTypes[props.formType]
```

Comment:
I wouldn't set a class attribute based on props, which are subject to change.

Reply:
Can it change? I was under the impression that once you login, you're viewing a specific form. You can't change forms during your session. A login has access to 1 type of form.

Reply:
Right, that is true bc of the business requirements of this app, however `props` by definition can change, so this pattern wouldn't be compatible with the expected lifecycle of a React component. Although it also seems like this (and all of the top level Section components) could be a pure functional component instead, which would make this a moot point.. what do you think?

Reply:
I don't think this would be an issue because the form type should never change even though props can. I thought of similar to initializing initial state and never changing it (because we shouldn't be). This way we'd be avoiding extra (even if they're cheap) operations.

However, I think you're right; these can be pure functional components.

Reply:
I've updated the component. Let me know your thoughts